cmd/validate_models.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/openfga/openfga/cmd/util"
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/storage/mysql"
+	"github.com/openfga/openfga/pkg/storage/postgres"
+	"github.com/openfga/openfga/pkg/storage/sqlcommon"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func NewValidateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "validate-models",
+		Short: "Validate authorization models. NOTE: this command is in beta and may be removed in future releases.",
+		Long:  "List all authorization models across all stores and run validations against them.\nNOTE: this command is in beta and may be removed in future releases.",
+		RunE:  runValidate,
+		Args:  cobra.NoArgs,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			flags := cmd.Flags()
+
+			util.MustBindPFlag(datastoreEngineFlag, flags.Lookup(datastoreEngineFlag))
+			util.MustBindPFlag(datastoreURIFlag, flags.Lookup(datastoreURIFlag))
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.String(datastoreEngineFlag, "", "the datastore engine")
+	flags.String(datastoreURIFlag, "", "the connection uri to the datastore")
+
+	// NOTE: if you add a new flag here, add the binding in PreRunE
+
+	return cmd
+}
+
+func runValidate(_ *cobra.Command, _ []string) error {
+	engine := viper.GetString(datastoreEngineFlag)
+	uri := viper.GetString(datastoreURIFlag)
+
+	ctx := context.Background()
+
+	var (
+		db  storage.OpenFGADatastore
+		err error
+	)
+	switch engine {
```

Comment:
Please keep the error messages surfaced from this switch consistent with the other code:
https://github.com/openfga/openfga/blob/main/cmd/run/run.go#L441-L460

Reply:
I'll send another PR that standardizes across the three commands